core-types/src/query/response/auth.rs

```diff
@@ -0,0 +1,3 @@
+pub use ibc_proto::cosmos::auth::v1beta1::QueryAccountResponse;
```

Comment:
Are you sure that we should re-export this structures?

Reply:
The highlighted struct has been imported. I've added new structs and they are used in grpc handlers. I may skip reexporting but I have to import them directly.